packages/suite/src/reducers/wallet/settingsReducer.ts

```diff
@@ -36,7 +37,10 @@ const settingsReducer = (state: State = initialState, action: Action): State =>
 
             case walletSettingsActions.changeNetworks.type: {
                 if (walletSettingsActions.changeNetworks.match(action)) {
-                    draft.enabledNetworks = action.payload;
+                    draft.enabledNetworks = [...action.payload].sort(
```

Comment:
Nit: `toSorted` is exactly for this purpose, [see here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) :slightly_smiling_face: 
```suggestion
                    draft.enabledNetworks = action.payload.toSorted(
```